#139 Fixed issues from "Added CameraViewActivity to store Student images"

schaemik opened the pull request.

Commit:
#139 fixxed issues;
- Added CameraViewActivity to the AndroidManifest
- Moved variable initialization to the top of the file

Commit:
Merge remote-tracking branch 'origin/master'

sladomic commented on app/src/main/java/org/literacyapp/authentication/CameraViewActivity.java:
```diff
@@ -31,15 +28,16 @@
+    private int count = 1;
 
+    // Image collection parameters
+    private static final Boolean diagnoseMode = true;
```
boolean instead of Boolean

schaemik replied:
Done

sladomic commented on app/src/main/java/org/literacyapp/authentication/CameraViewActivity.java:
```diff
@@ -31,15 +28,16 @@
     private String deviceId;
     private String collectionEventId;
     private StudentImageCollectionEventDao studentImageCollectionEventDao;
+    private int count = 1;
```
private int count;

jo-elimu replied:
What is the count for? Can we name the variable so that it explains what is being counted?

schaemik replied:
Yep, renamed to the imagesProcessed since it represented the images which have been checked for visible faces.

jo-elimu commented on app/src/main/AndroidManifest.xml:
```diff
@@ -53,6 +53,8 @@
             android:name=".task.StarActivity"
             android:screenOrientation="fullSensor" />
         <activity android:name=".task.LessonCompleteActivity"></activity>
+        <activity android:name=".authentication.CameraViewActivity">
```
Is this activity for _authentication_ or _training_, or both? I see it's in the "authentication" package.

schaemik replied:
Authentication stands in here for the complete set of functions which are needed for the face recognition. Starting with the process of taking images, storing them if faces are recognized, extracting features, start the training with the features and use the trained data to authenticate faces.

jo-elimu commented on app/src/main/java/org/literacyapp/authentication/CameraViewActivity.java:
```diff
@@ -31,15 +28,16 @@
 public class CameraViewActivity extends AppCompatActivity implements CameraBridgeViewBase.CvCameraViewListener2 {
```
Perhaps add a short JavaDoc paragraph explaining the responsibilities of this activity.

schaemik replied:
Alright

Commit:
#139 Fixed issues
- Added CameraViewActivity to AndroidManifest.xml
- Added JavaDoc to CameraViewActivity
- Renamed count variable to imagesProcessed
